circle.yml

```diff
@@ -0,0 +1,10 @@
+## Customize dependencies
+dependencies:
+  override:
+    - npm prune
```

Comment:
I guest you are using `npm prune` to get rid of old dependencies and allow `npm install` to get the latest one

I would recomend you use `npm update` instead: https://docs.npmjs.com/cli/update